test/spec/services/simulationService.js

```diff
@@ -56,16 +15,20 @@ describe('SimulationService', function () {
             });
         });
 
-        it('should return the complement from droitsDescription with the given droits', function() {
-            var actual = service.complement({ acs: { montant: 12 } });
-
-            expect(actual.acs).toBe(undefined);
-            expect(actual.apl.id).toBe('apl');
+        it('should sort eligible and non eligible droits', function() {
+            var openfiscaResult = { acs: 10 };
+            var droits = service.sortDroits(openfiscaResult);
+            expect(droits.droitsEligibles).toEqual({ acs: { id: 'acs', montant: 10 } });
+            expect(droits.droitsNonEligibles).toEqual({ apl: { id: 'apl' } });
+            expect(droits.droitsInjectees).toEqual({});
```

Comment:
So it would be better to have one big expect about droits instead of 3 ones ?
Because i don't think it would be relevant to split the 3 expects, there are not independent.